packages/go_router/lib/src/route.dart

```diff
@@ -996,6 +1011,7 @@ class StatefulShellBranch {
     this.initialLocation,
     this.restorationScopeId,
     this.observers,
+    this.preload = false,
```

Comment:
should we check for initiallocation is set if preload is true?

Reply:
Don't think that should be necessary, as it will fall back on the `defaultRoute` if `initialLocation` isn't provided.